controllers/release/release_suite_test.go

```diff
@@ -115,3 +118,26 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
 })
+
+// GetModuleCRDPath return the filesystem path after GOPATH/pkg/mod
+// for pseudo versioned modules.
+func GetModuleCRDPath(goModPath string, moduleName string) string {
```

Comment:
Can we make `goModPath` a constant based on `build.Default.GoPath`?

Also, move this function to a new package called `test` and a file called `paths.go`. The name of the function should probably be `GetRelativeDependencyPath` as this function has no relationship with CRDs. You would pass a `dependency string` which could be a full name or a substring (document it).

Reply:
the `goModPath` it the path to `go.mod`. The `build.Default.GoPath` points to `GOPATH`. Found that there is the `GOMOD` env. I will use this one instead.

Reply:
@davidmogar. The variable `GOMOD` that I was willing to use is not available unless you load the `go env`. So we have two options, or use the second function I added or add something like `source <$(go env)` so the `GOMOD` variable is availble, making it easyer to get the `go.mod` file path.

Reply:
Moved already to Makefile.